cli/src/main/java/org/openapitools/openapistylevalidator/cli/OptionManager.java

```diff
@@ -67,16 +71,35 @@ ValidatorParameters getOptionalValidatorParametersOrDefault(CommandLine commandL
         ValidatorParameters parameters = new ValidatorParameters();
         if (commandLine.hasOption(OPTIONS_OPT_SHORT)) {
             try {
-                Gson gson = new GsonBuilder().create();
                 String content = Utils.readFile(commandLine.getOptionValue(OPTIONS_OPT_SHORT), Charset.defaultCharset());
-                parameters = gson.fromJson(content, ValidatorParameters.class);
+                JsonParser parser = new JsonParser();
+                JsonElement jsonElement = parser.parse(content);
+                fixConventionRenaming(jsonElement, "path");
+                fixConventionRenaming(jsonElement, "parameter");
+                fixConventionRenaming(jsonElement, "property");
+                Gson gson = new GsonBuilder().create();
+                parameters = gson.fromJson(jsonElement, ValidatorParameters.class);
             } catch (Exception ignored) {
                 System.out.println("Invalid path to option files, using default.");
             }
         }
         return parameters;
     }
 
+    private void fixConventionRenaming(JsonElement jsonElement, String prefix) {
+        JsonObject jsonObject = jsonElement.getAsJsonObject();
+        String strategyKey = prefix + "NamingStrategy";
```

Comment:
Please use `String.format()` instead of using `+` for concatenation.

Reply:
Done